mergefs_test.go

```diff
@@ -1,21 +1,83 @@
 package mergefs_test
 
 import (
+	"io/fs"
+	"os"
+	"path/filepath"
 	"testing"
 	"testing/fstest"
 
 	"github.com/laher/mergefs"
+	"github.com/stretchr/testify/require"
```

Comment:
I can see the motivation for testify although I'd sooner use https://github.com/matryer/is

It's like `testify/assert` and `testify/require` in a single smaller package. It's got fewer features but cleaner IMO.

```
is := is.New(t)
if is.NoError(t, err) {
	is.Equal(t, len(dirs), fp.dirArrayLength)
}
```

Reply:
It's a bit outdated what you posted - f.ex. `t` not wanted now, `is.NoErr(err)` instead of `is.NoError(t, err)` etc.

Reply:
oh yup, my bad. Ta